packages/web3-react/src/hooks/useAccount.tsx

```diff
@@ -17,14 +17,18 @@ along with the library. If not, see <http://www.gnu.org/licenses/>.
 */
 import { getDefaultAlephiumWallet } from '@alephium/get-extension-wallet'
 import { useEffect } from 'react'
-import { useContext } from '../components/AlephiumConnect'
+import { useAlephiumConnectContext } from '../contexts/alephiumConnect'
 import { KeyType } from '@alephium/web3'
 
 export function useAccount(onDisconnected?: () => Promise<void>) {
-  const context = useContext()
+  const context = useAlephiumConnectContext()
 
   useEffect(() => {
     const handler = async () => {
+      if (context.connectorId === 'walletConnect') {
```

Comment:
Just to confirm: this means we don't support auto-connect for WalletConnect right now?

Reply:
Yes, because I'm not sure how we should support automatic connection via WalletConnect. I'll look into how to support this later.

Reply:
Np, I will assign the task to @ross-weir later since he has been looking into this for quit some time and got blocked by us for a while.